distribution/docs/src/main/resources/content/_reference/_dependencyList/ddf-dependency-list.adoc

```diff
@@ -126,9 +126,9 @@
 * org.apache.cxf:cxf-rt-transports-http:jar:3.2.9
 * org.apache.cxf:cxf-rt-ws-policy:jar:3.2.9
 * org.apache.cxf:cxf-rt-ws-security:jar:3.2.9
-* org.apache.felix:org.apache.felix.configadmin:jar:1.9.14
-* org.apache.felix:org.apache.felix.fileinstall:jar:3.6.4
-* org.apache.felix:org.apache.felix.framework:jar:5.6.12
```

Comment:
Ah yes, this file. Just at a glance, it's quite out-of-date. Not suggesting making any changes for this PR, but I wonder if this list is worth keeping around. It adds maintenance overhead but no real value imo

Reply:
I agree, I just made updates to versions I changed because I had seen you did the same with the last Karaf upgrade